### ISSUE TITLE
"my alexa skill" vs. "my skill"

### ISSUE THREAD

**Issue by bcichowlas** [alexa/interactive-adventure-game-tool#50]
Finally got it to work, thanks!  I'm hoping to use it in my elementary school programming class in some fashion.

I think there is an inconsistency in the tutorial section.  In the examples of using it, it says "launch my alexa skill", but in the development console screenshot it just says "my skill".

Being new at this, I kept saying "launch my alexa skill" and getting some vague response back.  It felt great when I said "launch my skill" and it worked!

**Comment by samdobson** [comment]
Good catch! I've updated the wording in my [fork](https://github.com/samdobson/interactive-adventure-game-tool) (as I'm not sure this version is still being maintained...)
